### PR TITLE
Native Button + Disable Button + Update Examples to ES6 syntax and react-native 0.24+

### PR DESCRIPTION
**Pull request by magrinj**
Hi, 

I create a new class called NativeButton, this class use `TouchableNativeFeedback` for Android apps, to get the smooth reaction on touch; for iOS that use `TouchableHighlight`.
This also allow to set a `disabledStyle`.

The `NativeButton` is also accessible with `Swipeout.NativeButton`.

I also remove the android example from the root of the projet, and move it to example folder.

Finally, I update the examples to `react-native 0.22.2` with ES6 syntax.

Hope this pull request can help :) 

Edit: I just add the ability to disable the button, with `disabled` props

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ left            | array  | Yes      | []        | swipeout buttons on left
 onOpen          | func   | Yes      |           |
 right           | array  | Yes      | []        | swipeout buttons on right
 scroll          | func   | Yes      |           | prevent parent scroll
+styles          | style  | Yes      |           | style of the container
```
Comment (KevinMartin) [inline README.md]: I think you meant `style` here. Typo?

**Comment by KevinMartin** [comment]
@dancormier do you think you can review this? Would love to see this merged in.

File: README.md (hunk 1 above)
Comment (magrinj) [inline README.md]: Yes sorry, I made a mistake..., I'll correct it!

**Comment by dmur** [comment]
In my opinion, this should be split up into several separate PRs to make it easier to review.

**Comment by dozoisch** [comment]
Looking forward for this!

**Comment by zimpower** [comment]
Have you guys seen the SwipeableListView and SwipeableRow in React Native / Experimental.
Are they alternatives to this library? Not sure how to use them directly.

**Comment by dozoisch** [comment]
@zimpower These component just got merged in master recently, the only doc they have is directly in the files.

**Comment by magrinj** [comment]
@zimpower I don't take attention to this new Class for the moment, I only hope this can be used without a ListView, if not I'm gonna stay on this lib :)

**Comment by tomauty** [comment]
Hey, I fixed the conflicts from this and will merge into master.

**Comment by magrinj** [comment]
Thanks for the merge!

**Pull request merged**